model/main.py

```diff
@@ -51,6 +67,13 @@
   except ImportError as e:
     raise e
 
+# If an argument like --countries=IN,US was passed, only process the listed
+# countries. This is helpful during development to speed up the processing.
+if args.contries:
+  other_config_files = [
```

Comment:
You could probably also do

```
other_config_files  = filter(lambda path: country_of_path(path) in args.contries, other_config_files)
```

Reply:
filter returns an iterable for lazy evaluation, so I have to serialize the result as a list:
```
  other_config_files = list(
      filter(lambda path: country_of_path(path) in args.contries,
             other_config_files))
```